<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CE5GoodsWaste</class>
 <widget class="QWidget" name="CE5GoodsWaste">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Code</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="C5LineEditWithSelector" name="lineEdit">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="Field" stdset="0">
      <string>f_id</string>
     </property>
     <property name="Primary" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Goods</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="C5LineEditWithSelector" name="leGoods">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="Field" stdset="0">
      <string>f_goods</string>
     </property>
     <property name="Type" stdset="0">
      <string>1</string>
     </property>
     <property name="First" stdset="0">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="C5LineEditWithSelector" name="leGoodsName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="FieldRef" stdset="0">
      <string>f_goodsname</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Reason</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="C5LineEditWithSelector" name="leReason">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="Field" stdset="0">
      <string>f_reason</string>
     </property>
     <property name="Type" stdset="0">
      <string>1</string>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="C5LineEditWithSelector" name="leReasonName">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="FieldRef" stdset="0">
      <string>f_reasonname</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Waste</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="C5LineEditWithSelector" name="lineEdit_4">
     <property name="maximumSize">
      <size>
       <width>100</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="Field" stdset="0">
      <string>f_waste</string>
     </property>
     <property name="Type" stdset="0">
      <string>2</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>C5LineEditWithSelector</class>
   <extends>QLineEdit</extends>
   <header location="global">c5lineeditwithselector.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
